hooks/emitter/decoder_test.go

```diff
@@ -174,7 +174,7 @@ func (suite *DecoderTestSuite) TestDecodeMsgExec() {
 	emitter.DecodeMsgExec(&msg, detail)
 	suite.testCompareJson(
 		detail,
-		"{\"grantee\":\"band1gaexzmn5v4jsqqqqqqqqqqqqqqqqqqqqwrdaed\",\"msgs\":[{\"amount\":[{\"denom\":\"uband\",\"amount\":\"1\"}],\"from_address\":\"band1gaexzmn5v4eqqqqqqqqqqqqqqqqqqqqq3urue8\",\"to_address\":\"band1gaexzmn5v4jsqqqqqqqqqqqqqqqqqqqqwrdaed\",\"type\":\"/cosmos.bank.v1beta1.MsgSend\"},{\"raw_reports\":null,\"request_id\":0,\"type\":\"/oracle.v1.MsgReportData\",\"validator\":\"\"}]}",
+		"{\"grantee\":\"band1gaexzmn5v4jsqqqqqqqqqqqqqqqqqqqqwrdaed\",\"msgs\":[{\"msg\":{\"amount\":[{\"denom\":\"uband\",\"amount\":\"1\"}],\"from_address\":\"band1gaexzmn5v4eqqqqqqqqqqqqqqqqqqqqq3urue8\",\"to_address\":\"band1gaexzmn5v4jsqqqqqqqqqqqqqqqqqqqqwrdaed\"},\"type\":\"/cosmos.bank.v1beta1.MsgSend\"},{\"msg\":{\"raw_reports\":null,\"request_id\":0,\"validator\":\"\"},\"type\":\"/oracle.v1.MsgReportData\"}]}",
```

Comment:
Check with P'Pang duay wa break pow

Reply:
just checked, and it'll break. 
Should we use the old structure?

Reply:
As discussed, we will until Cosmoscan is updated to support this.

Reply:
Hi @taobun , Cosmoscan team already updated to support this on testnet5 (https://laozi-testnet5.cosmoscan.io/tx/b380a36c01f536b91a603145e1d1061ad63f318aa15ab14f33c31ec77d9d6eb7).